src/Frontend/src/components/messages2/EditAndRetryButton.vue

```diff
@@ -19,8 +19,7 @@ const handleConfirm = async () => {
   isConfirmDialogVisible.value = false;
 
   const message = `Retrying the edited message ${state.value.data.id} ...`;
-  await showToastAfterOperation(store.retryMessage, TYPE.INFO, "Info", message);
```

Comment:
This is the actual fix, we no longer call `store.retryMessage` from here given that is already called in the dialog itself.